src/popup/router/components/Button.vue

```diff
@@ -43,9 +53,9 @@ export default {
     inline: Boolean,
     inactive: Boolean,
     to: { type: [String, Object], default: null },
-    bold: Boolean,
     backgroundless: Boolean,
     iconText: Boolean,
+    newUi: Boolean,
```

Comment:
I think the flag we used to use was `newUI`

Reply:
If I would set the prop as `newUI` then the prop in template should be `new-u-i` which in my opinion is rather bad. What do you think?

Reply:
You're right